toolkit/offchain/src/csl.rs

```diff
@@ -775,11 +775,22 @@ impl MultiAssetExt for MultiAsset {
 		asset: &AssetId,
 		amount: impl Into<BigNum>,
 	) -> Result<Self, JsError> {
-		self.set_asset(&asset.policy_id.0.into(), &asset.asset_name.to_csl()?, &0u64.into());
-		let mut assets = Assets::new();
-		assets.insert(&asset.asset_name.to_csl()?, &amount.into());
-		self.insert(&asset.policy_id.0.into(), &assets);
-		Ok(self)
+		let policy_id = asset.policy_id.0.into();
```

Comment:
there is similar cast above:
```
warning: casting to the same type is unnecessary (`u64` -> `u64`)
   --> toolkit/offchain/src/csl.rs:766:7
    |
766 |                     &(asset.amount as u64).into(),
    |                      ^^^^^^^^^^^^^^^^^^^^^ help: try: `asset.amount`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_cast
    = note: `#[warn(clippy::unnecessary_cast)]` on by default
```

Reply:
I've fixed all unnecessary conversions in this crate pointed by clippy